test/integration/xpack-limited.yml

```diff
@@ -0,0 +1 @@
+#no security
```

Comment:
Yes this file was placeholder for a new set of tests - x-pack without security. ill remove and repush.